wire-ios-system/WireSystem.xcodeproj/project.pbxproj

```diff
@@ -835,7 +835,6 @@
 				GENERATE_INFOPLIST_FILE = YES;
 				INFOPLIST_KEY_NSHumanReadableCopyright = "Copyright © 2024 Wire Swiss GmbH. All rights reserved.";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 15.0;
```

Comment:
Here and similar I remove unnecessarily overriding the `IPHONEOS_DEPLOYMENT_TARGET` values which have been set in config files, rather than updating this value